src/services/ApiService/History/types.ts

```diff
@@ -37,3 +38,11 @@ export interface ITxHistoryERC20Transfer {
   readonly contractAddress: TAddress;
   readonly amount: string; // Hex
 }
+
+export interface IFullTxHistoryValueTransfer {
+  readonly from: TAddress;
+  readonly to: TAddress;
+  readonly asset: Asset;
+  readonly amount: string; // Hex
+  readonly isNFTTransfer: boolean;
```

Comment:
Why is this needed if each asset has a type? Just check if `asset.type` is `ERC-20`, `ERC-721`, etc.